docs/getting-started/integrations/laravel.md

```diff
@@ -9,93 +9,106 @@ menu-title: Laravel
 
-To integrate CKEditor&nbsp;5 with Laravel, we will create a custom CKEditor&nbsp;5 build using the [online builder](https://ckeditor.com/ckeditor-5/online-builder/), and then import it into the Laravel project.
+## Using the Builder
 
-## Preparing a build
-
-In this guide, we will use the [online builder](https://ckeditor.com/ckeditor-5/online-builder/). It is a web interface that lets you create a custom build of CKEditor&nbsp;5 and download the code as a zip package.
-
-The online builder is a powerful tool that lets you effortlessly create a rich text editor that is custom-tailored to your needs. With the online builder, you can choose the desired editor type, and plugins, configure the toolbar, and choose the UI language for your editor.
-
-You can learn more about creating custom builds of CKEditor&nbsp;5 with the online builder in our {@link getting-started/legacy-getting-started/quick-start-other#creating-custom-builds-with-online-builder Customized installation} guide.
+The easiest way to use CKEditor&nbsp;5 in your Laravel project is preparing an editor preset with [Builder](https://ckeditor.com/builder?redirect=preset) and including it into your project.
 
 ## Setting up the project
 
-For the purpose of this guide, we will use a basic Laravel project created with [Composer](https://getcomposer.org/). You can refer to the [Laravel documentation](https://laravel.com/docs/10.x/installation) to learn how to set up a project in the framework.
-
-Once the project has been prepared, create an `assets/vendor` directory in the existing `public` directory in your app. Your folder structure should resemble this one:
-
-````
-├── app
-├── bootstrap
-├── config
-├── database
-├── public
-│   ├── assets
-|      └── vendor
-│   ├── .htaccess
-│   ├── favicon.ico
-│   ├── index.php
-│   ├── robots.txt
-│   └── webpack.config.js
-├── resources
-├── routes
-└── ...
-````
+This guide assumes you have a Laravel project. You can create a basic Laravel project using [Composer](https://getcomposer.org/). Refer to the [Laravel documentation](https://laravel.com/docs/10.x/installation) to learn how to set up a project in the framework.
 
-## Integrating the build in your Laravel project
+## Integrating from CDN
 
-Once you have your custom editor build ready and the Laravel project has been set up, extract the `.zip` folder obtained from the online builder and place it in the `assets/vendor` directory created in the previous step. Your folder structure should now look like this:
+Once the project has been prepared, create an `assets/vendor/ckeditor5.js` file in the existing `public` directory in your app. Your folder structure should resemble this one:
 
-````
+````plain
 ├── app
 ├── bootstrap
 ├── config
 ├── database
 ├── public
 │   ├── assets
 |      ├── vendor
-|          └── ckeditor5
+|          └── ckeditor.js
```

Comment:
This whole block is not needed. File will be downloaded via CDN script (as below).

Reply:
In the current form of this guide - it is needed. I recommended putting there the code from the Builder (JS snippet). Scripts from the snippet below will download the editor and styles, but we also need to create the editor in our project.

I could put everything in one HTML file. I tested it, and it works. But it looked cluttered to me.

Reply:
Gotcha, sorry missed that! You're right!